config/settings.yml

```diff
@@ -21,8 +21,10 @@ hesa:
   auth_url: "https://identity.hesa.ac.uk/Account/RemoteLogOn?ReturnDomain=https://datacollection.hesa.ac.uk&ReturnUrl=%2f"
   collection_base_url: "https://datacollection.hesa.ac.uk/apis/itt/1.1/CensusData"
   trn_data_base_url: "https://datacollection.hesa.ac.uk/apis/itt/1.1/TRNData"
+  trn_file_base_url: "https://datacollection.hesa.ac.uk/apis/itt/1.1/TRNFile"
   current_collection_reference: "C21053"
   current_collection_start_date: "2022-04-01"
+  trn_file_upload_start_date: TBC
```

Comment:
We need to determine a date before merging.

Reply:
As above, maybe we don't need the date. Let's discuss.